src/common/transformations/src/transformations/sdpa_to_paged_attention/state_management_pattern.cpp

```diff
@@ -231,27 +239,31 @@ ov::pass::StateManagementPattern::StateManagementPattern(ParameterVector& kv_par
             std::make_shared<v1::Reshape>(q_transpose, v0::Constant::create(element::i64, Shape{2}, {0, -1}), true);
 
         ov::Output<ov::Node> k_target_layout, v_target_layout;
-        if(pattern_map.count(qkv_current_split_node)) {
+        if (pattern_map.count(qkv_current_split_node)) {
             std::cerr << "[ DEBUG ] kv_concat_split detected\n";
```

Comment:
can we move all such debug prints under macro, which can be defined in this file during development?
or under env var

Reply:
Moved under /dev/null.